tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -1467,6 +1571,155 @@ public function test_intent_error_throws_exception() {
+		$this->mock_gateway->expects( $this->once() )
+			->method( 'stripe_request' )
+			->with( "payment_intents/$payment_intent_id?expand[]=payment_method" )
+			->willReturn( $this->array_to_object( $payment_intent_mock ) );
+
+		// Intercept wp_safe_redirect so that exit() is never reached, allowing assertions to run.
+		$redirect_url = null;
+		add_filter(
+			'wp_redirect',
+			function ( $location ) use ( &$redirect_url ) {
+				$redirect_url = $location;
+				throw new \RuntimeException( 'redirect_intercepted' );
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '1698', column '15'). (undefined)

(MissingImport)